packages/protobuf-example/package.json

```diff
@@ -3,10 +3,9 @@
   "private": true,
   "license": "(Apache-2.0 AND BSD-3-Clause)",
   "scripts": {
-    "clean": "rm -rf ./dist/esm/* ./dist/types/*",
-    "build": "../../node_modules/typescript/bin/tsc --project tsconfig.json --module ES2015 --verbatimModuleSyntax --outDir ./dist/esm --declaration --declarationDir ./dist/types",
-    "add-person": "node dist/esm/add-person.js addressbook.bin",
-    "list-people": "node dist/esm/list-people.js addressbook.bin",
+    "build": "../../node_modules/typescript/bin/tsc --noEmit",
+    "add-person": "tsx src/add-person.ts addressbook.bin",
+    "list-people": "tsx src/list-people.ts addressbook.bin",
```

Comment:
A drive-by change. We have been using `tsx` in our examples everywhere but here.